internal/pkg/describe/rd_web_service.go

```diff
@@ -151,6 +195,20 @@ func (w *rdWebSvcDesc) HumanString() string {
 	fmt.Fprint(writer, color.Bold.Sprint("\nConfigurations\n\n"))
 	writer.Flush()
 	w.AppRunnerConfigurations.humanString(writer)
+	if w.hasObservabilityConfiguration() {
+		fmt.Fprint(writer, color.Bold.Sprint("\nObservability\n\n"))
+		writer.Flush()
+		headers := []string{"Environment", "Tracing"}
+		fmt.Fprintf(writer, "  %s\n", strings.Join(headers, "\t"))
+		fmt.Fprintf(writer, "  %s\n", strings.Join(underline(headers), "\t"))
+		for _, config := range w.Observability {
+			tracingVendor := "None"
+			if config.Tracing != nil && config.Tracing.Vendor != "" {
+				tracingVendor = config.Tracing.Vendor
+			}
+			fmt.Fprintf(writer, "  %s\t%s\n", config.Environment, tracingVendor)
+		}
+	}
```

Comment:
I wonder if we could have all this logic as a method on the `Observalibility` type? 

```go
func (o *Observability) render(w io.Writer) 
```
so that with Danny's ECS changes we can have all workload types populate the `Observability` struct and then call this method to render which should reduce code duplication